deploy/pathplanner/2bL-hangerbounce-a.path

```diff
@@ -0,0 +1 @@
+{"waypoints":[{"anchorPoint":{"x":6.1,"y":5.1},"prevControl":null,"nextControl":{"x":5.718034679107333,"y":5.5051147342801015},"holonomicAngle":135.0,"isReversal":false,"velOverride":null,"isLocked":false},{"anchorPoint":{"x":5.2861431665298895,"y":5.97428902088289},"prevControl":{"x":5.660031573232004,"y":5.752629226770399},"nextControl":null,"holonomicAngle":160.80995375313864,"isReversal":false,"velOverride":null,"isLocked":false}],"maxVelocity":4.95,"maxAcceleration":4.95,"isReversed":null}
```

Comment:
Any thoughts on running prettier on these?  I know they are auto generated but it would also reviewing diffs in the future, when its a single line that is basically impossible.  I’m not sure how consistent pathplanner is with respect to generating the same output every time though.

Reply:
I'm guessing either it has to be one line to be used or that if formatted, the program could no longer update it. These are a couple thousand characters long each so it's not worth trying to read through them. The ideal way is to use path planner and open the project so that you can see each trajectory